pkg/cnab/provider/docker_linux.go

```diff
@@ -47,3 +47,14 @@ func (r *Runtime) mountDockerSocket(cfg *container.Config, hostCfg *container.Ho
 	hostCfg.Mounts = append(hostCfg.Mounts, dockerSockMount)
 	return nil
 }
+
+func (r *Runtime) addVolumeMountToHostConfig(hostConfig *container.HostConfig, source string, target string, readOnly bool) error {
+	mount := mount.Mount{
+		Source:   source,
+		Target:   target,
+		Type:     "bind",
```

Comment:
Should this be volume instead of bind?

Reply:
I did try with volume, however then you cannot mount a target folder where the name of the folder starts with a dot, the error received is something like `cannot create container: Error response from daemon: create /abc/.xyz: "/Users/testuser/.xyx" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed`